keycloak/keycloak
# Fix: Display Labels for Selected Items in Multiselect User Profile Attributes

### Description:
This pull request addresses [Issue #34456](https://github.com/keycloak/keycloak/issues/34456) where multiselect attributes in the Keycloak user profile displayed raw values instead of the configured labels for selected items.
Solves 

**Problem:**
When using a multiselect attribute with labels configured in the user profile, labels were correctly rendered in the dropdown options. However, after selecting an option, the component displayed the raw values for each selected item, creating an inconsistent and confusing experience for users.

**Solution:**
The fix ensures that the labels are applied to selected items as well, so both the dropdown options and the selected items display the configured labels consistently.

**Steps to Verify:**

- Configure a user profile attribute with type multiselect, including multiple options with labels.
- Open the user profile page containing this attribute.
- Verify that both the dropdown options and selected items display the labels, not the raw values.
